docs/source/docker/docker.rst

```diff
@@ -0,0 +1,100 @@
+==================================================
+NI Linux Real-Time and Docker
+==================================================
+
+.. contents:: Table of Contents
+   :depth: 2
+   :local: 
+
+Introduction
+============
+
+The `Docker platform`_ helps with producing consistent environments and isolating the processes
+within them.
+Docker can be used on NI Linux Real-Time with NI hardware for test and measurement use cases.
+The document encompasses the basics of how to install Docker and run containers from an
+NI Linux Real-Time system.
+
+.. _Docker platform: https://www.docker.com
+
+Installation
+============
+
+To install Docker on an NI Linux Real-Time system, simply run the following command
+from your target:
+
+.. code:: bash
+    
+    opkg install docker-ce
+
+.. note::
+
+    Currently Docker will not function correctly if you have LabVIEW RT installed.
```

Comment:
Is there a workaround for this that we should list? Or is the only workaround to not install LV RT?

Reply:
From what I understand, we'll need LabVIEW to start using cgroups v2 (see AB#2326685 and associated wiki page). I don't recall if we have a workaround that gets both working simultaneously.

Reply:
If I'm not mistaken a work around is currently in development, but it's unclear when it will be complete, so it was determined that we put this note in for now, and it will be removed when we have them working together

Reply:
That makes sense to me. I think we can leave this as is, then.